gluster-exporter/metric_brick.go

```diff
@@ -128,6 +179,160 @@ func diskUsage(path string) (disk DiskStatus, err error) {
 	return
 }
 
+// LVMStat represents LVM details
+type LVMStat struct {
+	Device          string
+	UUID            string
+	Name            string
+	DataPercent     float64
+	PoolLV          string
+	Attr            string
+	Size            float64
+	Path            string
+	MetadataSize    float64
+	MetadataPercent float64
+	VGName          string
+}
+
+func getLVS() ([]LVMStat, error) {
+	cmd := "lvm vgs --unquoted --noheading --nameprefixes --separator $ --nosuffix --units m -o lv_uuid,lv_name,data_percent,pool_lv,lv_attr,lv_size,lv_path,lv_metadata_size,metadata_percent,vg_name"
```

Comment:
Is `$` a legal character in any of the requested names? If so, this needs a different approach.

Reply:
I can think of using `|` rather. Mostly `|` is not a valid character for names I feel. What you suggest?

Reply:
I think there are potential parsing errors no matter the separator. Perhaps using a standard json parser and `--reportformat=json` would be more robust.

Reply:
Thats a good suggestion. Will use that.